ee/codegen/src/utils/uuid.ts

```diff
@@ -0,0 +1,8 @@
+import { python } from "@fern-api/python-ast";
+import { validate as uuidValidate } from "uuid";
+
+export function generateId(id: string): python.TypeInstantiation {
```

Comment:
nit: Given that this returns an AstNode, it seems reasonable for this to be a class called something like `UuidOrString` and extend `AstNode` directly. This would better follow the pattern's we've established so far.